etc/schemas/2022/01/01/FMKResponseHeader.xsd

```diff
@@ -25,7 +25,7 @@
 			<element name="Paging" type="mc160:PagingResponseType" minOccurs="0"/>
 			<element name="Warnings" type="mc160:FMKDetailType" minOccurs="0" maxOccurs="unbounded" />
 			<!-- TODO: FMKConfiguration -->
-
+			<any minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
Den her hyler leverandørerne tit over da deres kodegenereringstools åbenbart har det svært med "any"  elementer. jeg er helt med på det nyttige af det for os, men lad os lige overveje, hvor sikre/usikre vi er på indeholdet af response-headerne. Hvis vi er ret sikre på det, kan any måske undgås, og så kan vi versionere headerne sammen med evt. extensions af selve snitfladen?